homeassistant/components/sensor/yr.py

```diff
@@ -139,7 +140,7 @@ class YrData(object):
 
     def __init__(self, hass, coordinates, devices):
         """Initialize the data object."""
-        self._url = 'http://api.yr.no/weatherapi/locationforecast/1.9/'
+        self._url = 'http://api.met.no/weatherapi/locationforecast/1.9/'
```

Comment:
Can you update host to `https://aa015h6buqvih86i1.api.met.no`, that's a unique url for HA that they generated to better track usage.